src/novo/src/server.ts

```diff
@@ -113,14 +117,18 @@ function initializeNovo() {
                 "/assets-novo/novo-artsy.js"
               )}"></script>
 
-              <div id='react-root'>
-              ${bodyHTML}
-              </div>
+              <div id="react-modal-container"></div>
+              <div id='react-root'>${bodyHTML}</div>
 
               ${scripts}
+              <style type="text/css">
+              .grecaptcha-badge { visibility: hidden; }
+              </style>
               <script src="${novoManifest.lookup(
                 "/assets-novo/novo-artsy-novo.js"
               )}"></script>
+              <!-- TODO: add eigen exclude -->
```

Comment:
Still need to exclude this script from loading in eigen, can use var `sd.EIGEN`

Reply:
I would prefer to wait until the current HTML Webpack plugin task is in place. The template will require redoing this work from scratch.

Reply:
https://github.com/artsy/force/pull/6741/files#diff-8e91e6a62be6cbd001214e39acfed4a04d076583afcc37ef3152e5f5cfca1c65R25